net-mgmt/pfSense-pkg-snmptt/files/usr/local/pkg/snmptt.inc

```diff
@@ -0,0 +1,136 @@
+<?php
+/*
+ * snmptt.inc
+ *
+ * part of pfSense (https://www.pfsense.org)
+ * Copyright (c) 2017 Rubicon Communications, LLC (Netgate)
+ * All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+require_once("util.inc");
+require_once("functions.inc");
+require_once("pkg-utils.inc");
+require_once("globals.inc");
+
+define('SNMPTT_BASE', '/usr/local');
```

Comment:
These kind of constant definition doesn't make much sense after we moved to pkg. It was used before because of PBIs. If something like this is used nowadays it would need to respect ${PREFIX} from Makefile but I don't believe it's worth the effort.